flats/flatsscraper/flatsscraper/spiders/flatsspiders.py

```diff
@@ -0,0 +1,30 @@
+import scrapy
+from flatsscraper.items import FlatItem
+import re
+
+
+class FlatsspidersSpider(scrapy.Spider):
+    name = "flatsspiders"
+    allowed_domains = ["dom.trojmiasto.pl"]
+    start_urls = ["https://dom.trojmiasto.pl/nieruchomosci-rynek-wtorny/e1i,27_60_17_14_13_18_16_24_19_25_6_15_61_20_21,ii,1,qi,45_,wi,100.html"]
+
+    def parse(self, response):
+        flats = response.css(".ogloszeniaList__item")
+        #flats = response.css(".ogloszeniaList.ogloszeniaList__item")
+        for flat in flats:
+            item = FlatItem()
+            try:
+                item['price'] = int(re.sub("[^0-9]", "", flat.css("p.ogloszeniaList__price::text").get()))
+                item['url'] = flat.css("a.ogloszeniaList__img").attrib["href"]
+                item['numbers_of_rooms'] = flat.css(".ogloszeniaList__detail.button.button--label.button--"
```

Comment:
Tutaj dane mają skrajnie białe znaki po zescrapowaniu. Trzbeba to usunąć.

Reply:
ok